src/pds/registry/utils/titan_treks/create_titan_treks_pds4.py

```diff
@@ -0,0 +1,79 @@
+"""Script to build pds4 xml for Titan Treks OGC WMTS layers.
+
+The layers are scraped using the Titan Treks API.
+"""
+import argparse
+from pathlib import Path
+
+import requests
+
+from . import product_service_builder as psb
+
+
+def main():
+    """Main function of script."""
+    # set up command line args
+    parser = argparse.ArgumentParser(description="Customize save pds4 xml labels created for Titan Treks",
```

Comment:
I am not sure I understand this description. Should we say something like:
"Generate PDS4 XML labels for Titan Treks OGC/WMTS GIS Service"